src/main/java/medicaldoctor/controllers/AdditionalLoginController.java

```diff
@@ -0,0 +1,60 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.PasswordField;
+import medicaldoctor.backend.LoginService;
+import medicaldoctor.backend.data.LoginResult;
+import medicaldoctor.core.AppSession;
+import medicaldoctor.entities.UserType;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public class AdditionalLoginController implements Initializable, ParentController{
+    ControllerManager cm;
+    
+    @FXML
+    private Button loginButton;
+    @FXML
+    private PasswordField password;
+    @FXML
+    private Label i_label;
+    
+    @FXML
+    void loginButtonClicked(ActionEvent event) throws Exception{
+        LoginResult result = LoginService.checkAdditionalPassword(password.getText());
+        if(result == LoginResult.SUCCESS){
+            if(AppSession.getActiveUser().getType().equals(UserType.ADMIN)){
+                cm.load(LookUp.ADMIN_DASHBOARD_NAME, LookUp.ADMIN_DASHBOARD);
```

Comment:
again, i think there should be a way that only requires one identifier.

Reply:
This is easier for me to understand. It makes sense, but if you really prefer a different method I can find one.

Reply:
you dont need to find one. you just use the `LookUp.ADMIN_DASHBOARD`, which is already a  unique string, as the map key instead of the `LookUp.ADMIN_DASHBOARD_NAME`